_pages/jobs.md

```diff
@@ -8,3 +8,5 @@ author_profile: true
 - 3 April 2022: PhD grant on [Compositional Verification of Smart Contracts in Isabelle](https://www.exeter.ac.uk/study/funding/award/?id=4326), University of Exeter, UK, deadline: 29 April 2022
 
 - 3 April 2022: PhD grant on [Formal Verification for Safety- or Security-Critical Systems](https://www.exeter.ac.uk/study/funding/award/?id=4328), University of Exeter, UK, deadline: 29 April 2022
+
+- 8 May 2022: PhD and Postdoc positions on [Satisfiability Modulo Theories](https://u.cs.biu.ac.il/~zoharyo1/jobs.txt), Bar-Ilan University, Israel. Start date: 2022 or 2023.
```

Comment:
Hi @yoni206 . Could you please add your entry on the top so that the newest comes first? Best regards, Frédéric.

Reply:
Thanks for the review, done. 
[ed37e5a](https://github.com/EuroProofNet/europroofnet.github.io/pull/14/commits/ed37e5aacbcd5586a1e3a70c9ef43a766a64c276)